python/bunsen/mapping/__init__.py

```diff
@@ -10,41 +10,59 @@ class for details.
 import collections
 import datetime
 
-def get_default_concept_maps(spark_session):
+def get_concept_maps(spark_session, database='ontologies'):
+    """
+    Returns a :class:`ConceptMaps` instance for the given database.
```

Comment:
Should this docstring include `:py` as a prefix to the `:class:ConceptMaps` reference?

Reply:
I had thought the same, but it seems that the prefix is implied when in a doc string. I did verify the links showed up appropriately in the generated documentation.